### ISSUE TITLE
Interesting title, can you provide more information, such as code and paper?

### ISSUE THREAD

**Issue by lichen14** [apple1986/SAMatch#1]
Interesting title, can you provide more information, such as code and paper?

**Comment by apple1986** [comment]
Thanks for your attention. Please wait a few days. 




At 2024-10-28 17:49:32, "lichen14" ***@***.***> wrote:

Interesting title, can you provide more information, such as code and paper?

**Comment by apple1986** [comment]
> Interesting title, can you provide more information, such as code and paper?

You can downloaded the paper in https://arxiv.org/abs/2411.16949 . Thanks for your attention.

**Comment by apple1986** [comment]
Hi, the paper has been attached. Thanks for your attention. 




https://arxiv.org/abs/2411.16949










在 2024-10-28 22:43:20，"Guoping Xu" ***@***.***> 写道：

Thanks for your attention. Please wait a few days. 




At 2024-10-28 17:49:32, "lichen14" ***@***.***> wrote:

Interesting title, can you provide more information, such as code and paper?

**Comment by lichen14** [comment]
Hi,thanks for the reply. 

It seems the proposed SAMatch is heavily based on the Match-based network, such as FixMatch and UniMatch. 

So is the project implemented on FixMatch and UniMatch?  Or is there any plan for releasing the code and training logs?

**Comment by apple1986** [comment]
Yes, it is mainly based on the FixMatch and UniMatch. The project is implemented on UniMatch (FixMatch) and SAM (MedSAM). You can refer these project firstly. 

While your paper is under review, it seems that the code might take some time to be released. Thanks for your interesting.










At 2024-11-30 21:12:12, "lichen14" ***@***.***> wrote:

Hi,thanks for the reply.

It seems the proposed SAMatch is heavily based on the Match-based network, such as FixMatch and UniMatch.

So is the project implemented on FixMatch and UniMatch? Or is there any plan for releasing the code and training logs?
